src/PolyhedralGeometry/iterators.jl

```diff
@@ -251,3 +259,16 @@ function halfspace_matrix_pair(iter::SubObjectIterator)
         throw(ArgumentError("Halfspace-Matrix-Pair not defined in this context."))
     end
 end
+
+Polymake.convert_to_pm_type(::Type{SubObjectIterator{RayVector{T}}}) where T = Polymake.Matrix{T}
+Polymake.convert_to_pm_type(::Type{SubObjectIterator{PointVector{T}}}) where T = Polymake.Matrix{T}
+Base.convert(::Type{<:Polymake.Matrix}, iter::SubObjectIterator) = matrix_for_polymake(iter; homogenized=true)
+
+function homogenized_matrix(x::SubObjectIterator{<:PointVector}, v::Number = 1)
+    @assert v == 1
```

Comment:
What's the point of this argument `v` if it must always be 1 anyway? Is it meant to be some kind of additional type/error checking?

Reply:
The argument is for compatibility with another `homogenized_matrix` function which explicitly does the homogenization for matrices using that value as a first coordinate (instead of the iterators here where we want to fetch an already homogenized matrix from the polymake object).
But this should probably be turned into a more helpful exception message via `throw` which mentions what to do in case of the error.

Reply:
OK that makes sense, thanks for the explanation